airbyte-commons-worker/src/main/java/io/airbyte/workers/internal/NamespacingMapper.java

```diff
@@ -59,9 +59,9 @@ public ConfiguredAirbyteCatalog mapCatalog(final ConfiguredAirbyteCatalog inputC
   }
 
   @Override
-  public AirbyteMessage mapMessage(final AirbyteMessage inputMessage) {
-    if (inputMessage.getType() == Type.RECORD) {
-      final AirbyteMessage message = Jsons.clone(inputMessage);
```

Comment:
why do we do this? this is very expensive?

in theory, just changing the field shouldn't need a clone